FOR-DEVS.md

```diff
@@ -58,14 +58,16 @@ If you are part of theCourseForum engineering team, follow the instructions belo
     ```
 3. Copy the [project secret keys](https://docs.google.com/document/d/1HsuJOf-5oZljQK_k02CQhFbqw1q-pD_1-mExvyC1TV0/edit?usp=sharing) into a `.env` file in the project base directory `theCourseForum2/`
 4. In the `theCourseForum2/` directory, run `docker-compose up` to start your Docker containers (tCF Django app and database)
-5. Download a copy of the database from [Google Drive](https://drive.google.com/open?id=1ubiiOj-jfzoBKaMK6pFEkFXdSqMuD-22)
+5. Download a copy of the database from [Google Drive](https://drive.google.com/file/d/1S9GNFydvw6ZZW0AGHy6zyTLl-pvyIT8Q/view?usp=sharing)
     - put this into the base `theCourseForum2/` directory
-6. While your container is still running, open a second terminal, cd into `theCourseForum2/`, and run this script to set up your database*:
-    - `./load_data ${DB}` 
-    - where `${DB}` should be replaced with the latest DB file (e.g. `./load_data april7.sql`)
+6. Once the Django server is done loading (i.e. when you see `tcf_django | Watching for file changes with StatReloader`), open a second terminal, cd into `theCourseForum2/`, and run the following two lines of script to set up your database, one at a time:
+    ```
+    docker exec -i tcf_db psql tcf_db -U tcf_django -c "DROP SCHEMA public CASCADE; CREATE SCHEMA public;"
```

Comment:
did you rename the database to `public`?

Reply:
@j-alicia-long  No, I checked and all our tables belong to a single schema named `public`, which I think is the default value. I believe there are multiple schemas in a single database and multiple tables in a single schema. I found this method here: https://stackoverflow.com/questions/3327312/how-can-i-drop-all-the-tables-in-a-postgresql-database.
This is what you see when you run `docker exec -ti tcf_db psql tcf_db -U tcf_django -c "select * from pg_tables;"`:
![image](https://user-images.githubusercontent.com/7676354/100802525-f9621880-346c-11eb-9db7-690b90df57d1.png)
Seemed like a good trick for us.

Reply:
@Bartleby2718 Hmm that's interesting since our DB setup/reset docs says to drop & recreate the `tcf_db` database?
![image](https://user-images.githubusercontent.com/4724192/100826747-0f4dea00-3429-11eb-9151-df155dae6917.png)

Reply:
@j-alicia-long Just so we're on the same page, the hierarchy looks like this (`1---*` means one-to-many relationship):
> PostgreSQL server running on the `tcf_db` container
> - 1---* database
>   - 1---* schema
>      - 1---* table

Should I have written `schema` in the doc? 🤔